src/bashly.yml

```diff
@@ -42,6 +41,26 @@ commands:
         required: true
     examples:
       - dcu logo coreboot.rom -l bootsplash.bmp
+  - name: smmstore
+    help: Edit settings stored in SMMSTORE
```

Comment:
Perhaps `Dasharo UEFI runtime variables` ? @SergiiDmytruk @krystian-hebel ideas?

`SMMSTORE` may be a rather vague concept for an end user

Reply:
Technically, at this level this has to do with neither UEFI runtime nor SMM. Maybe keep it as simple as possible, e.g. `Edit Dasharo settings`?

Reply:
I was thinking on distinguishing with another variants of Dasharo, possibly in the future. But even then, we can probablyhave a common option, and detect the fw variant as necessary.

Reply:
Even without automatic detection should be possible to add a flag to specify kind of a setting in case of need.  If end user doesn't care how things are stored, I agree that it shouldn't be exposed provided that it can actually be kept hidden.

Reply:
Changed to `Edit Dasharo configuration variables` in https://github.com/Dasharo/dcu/pull/5/commits/2cf531f645fc532b64383b6104c3be1b6fba0c75